test/counts.jl

```diff
@@ -90,6 +90,17 @@ pm = proportionmap(x)
 xx = repeat([6, 1, 3, 1], outer=100_000)
 cm = countmap(xx)
 @test cm == Dict(1 => 200_000, 3 => 100_000, 6 => 100_000)
+#with iterator
+xx_missing = skipmissing(xx)
+cm_missing = countmap(xx_missing)
+@test cm_missing isa Dict{Int, Int}
+@test cm_missing == cm
+
+cm_any_itr = countmap((i for i in xx)) 
+@test cm_any_itr[1] == 200_000
+@test cm_any_itr[3] == 100_000
+@test cm_any_itr[6] == 100_000
+@test cm_any_itr isa Dict{Any,Int} #no knowledge about type
```

Comment:
Would this work too?
```suggestion
@test cm_any_itr isa Dict{Any,Int} # no knowledge about type
@test cm_missing == cm
```

Reply:
no because `cm`'s key has type info, so not `Any`?

Reply:
nvm, it does work